### ISSUE TITLE
Feature request: run commands on host when connect via ssh

### ISSUE THREAD

**Issue by jereze** [jakob/Postico#726]
When Postico opens a connection via a ssh connect, it would be nice to have the ability to run first a command on the host.

My use case is to start first the proxy on a Kubernetes cluster: `kubectl port-forward the-pod-with-postgresql 7070:5432 -n namespace`

**Comment by jakob** [comment]
This sounds useful, but possibly a bit tricky to get right.

I'm not familiar with Kubernetes. In this example, what should happen when Postico opens multiple connections? Should the command be executed for each connection?

**Comment by jereze** [comment]
Yes, Postico can open multiple connections through a single execution of the `kubectl port-forward` command. But I guess we can also accept to be configured at the connection level (in Postico) and not to support multiple connection (since the command will fail if run twice in parallel).

Also, let me make it clear, I need to run this `kubectl port-forward` command on a remote server and not locally because we use a "bastion" host. In other scenario, running this command locally may be completely fine.
